homeassistant/components/go2rtc/__init__.py

```diff
@@ -46,22 +53,47 @@
 )
 
 
-async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
-    """Set up WebRTC from a config entry."""
-    if binary := entry.data.get(CONF_BINARY):
+CONFIG_SCHEMA = vol.Schema({DOMAIN: {vol.Optional(CONF_URL): cv.url}})
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up WebRTC."""
+    url: str | None = None
+    if DOMAIN not in config or not (url := config[DOMAIN].get(CONF_URL)):
```

Comment:
How will the integration be setup without the domain in the config, if there's no config flow?

Reply:
The idea was to add go2rtc to `default_config` in a follow-up. I'll remove the check for `DOMAIN` until then